config/config.js

```diff
@@ -0,0 +1,32 @@
+require("dotenv").config({
+  path: __dirname + "/../.env"
+});
+module.exports = {
+  development: {
+    username: process.env.DB_USERNAME,
+    password: process.env.DB_PASSWORD,
+    database: process.env.DB_DATABASE,
+    host: process.env.DB_HOST,
+    dialect: "postgres",
+    define: {
+      underscored: true,
+      createdAt: "created_at",
+      updatedAt: "updated_at",
+      deletedAt: "deleted_at",
+    },
+  },
+  test: {
+    username: 'root',
+    password: null,
+    database: 'database_development',
+    host: '127.0.0.1',
+    dialect: 'postgres',
+  },
+  production: {
+    username: "root",
+    password: null,
+    database: "database_production",
+    host: "127.0.0.1",
+    dialect: "mysql",
+  },
```

Comment:
use environment variable from .env for production as well.

Reply:
done.